app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="circle_size">72dp</dimen>
```

Comment:
Given you've added `first_circle_size` and `second_circle_size`, I think it would make more sense for this one to be renamed as `drag_circle_size`.